fbw-a32nx/src/base/flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/aircraft.cfg

```diff
@@ -86,11 +86,10 @@ ui_certified_ceiling = 39800 ; service ceiling / max certified operating altitud
 ui_max_range = 3500 ; max distance the aircraft can fly between take-off and landing in (NM)
 ui_autonomy = 7 ; max duration the aircraft can fly between take-off and landing in (Hrs)
 ui_fuel_burn_rate = 5300 ; average fuel consumption per hour (lbs/hr) - reminder: fuel density is ~6.7lbs per US gallon
-atc_id = "ASX320" ; tail number
-atc_id_enable = 0 ; enable tail number
-atc_airline = "" ; airline name
-atc_flight_number = "1123" ; flight number
-atc_heavy = 1 ; heavy?
+atc_id = "FBW320" ; tail number
+atc_airline = "FBWSIM" ; airline name
```

Comment:
How does this sound in the sim TTS?

Reply:
no idea :D :D - I don't use ingame ATC. Should we rather leave it?

Reply:
We should probably remove this and not define atc_id explicitly at the base level. We should leave it to livery to define atc_id (for example currently the house livery currently has atc_id: "A32NX" defined there)

Reply:
in a sense this is the default livery so it should be defined I guess, right?